doc/source/getting_started/write_documentation.rst

```diff
@@ -26,7 +26,7 @@ documentation dependencies with this command:
 
 .. code:: console
 
-    pip install -e '.[doc]'
+    pip install -e .[doc]
```

Comment:
`zsh` terminal (MacOS) requires those ticks, and it is still compatible with `bash`... so I rather keep them.

Reply:
This is not compatible with CMD though AFAIK

Reply:
My bad, I thought it was a typo as I use Windows

Reply:
I will delete the change as this is not the purpose of the PR